.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  pull_request:
+  push:
```

Comment:
Is this the syntax for building on pull requests with default configuration?

Reply:
Yeah